packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -578,12 +578,11 @@ const Messages: FunctionComponent<MessagesProps> = ({
     updateFieldState("threadDeletingConfirmation", true)
   }
 
-  const handleDeleteTmpThreadClick = async (): Promise<void> => {
+  const handleDeleteTmpThreadClick = (): void => {
     if (tmpActiveThread !== undefined) {
       closeSidebars()
     } else if (activeThread) {
-      await deleteThreads([activeThread.id])
-      resetItems()
```

Comment:
Not 100%, but is it ok that `resetItems()` is removed?

Reply:
`resetItems` is inside added `handleDeleteThread ` so it will be still there ;)